syntax = "proto3";

import "google/protobuf/timestamp.proto";

package hoppipolla.common;

message Interval {
  google.protobuf.Timestamp start_time = 1;
  google.protobuf.Timestamp end_time = 2;
}
